CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Changelog for NeoFS Node
 - TLS RPC support for IR consensus mode (#2322)
 
 ### Changed
+- NeoFS CLI generates random private key automatically if wallet is omitted (#2123)
```

Comment:
for all the commands? even `object put`?

Reply:
for those that use `key.GetOrGenerate`. There are several command using `key.Get` which doesn't fallback to random key (e.g. `container put` can't exec w/o wallet obviously)

Reply:
oh, i see. containers are safe but `object put/delete`, `container set-ecl` are not. well do not totally agree but not that critical

Reply:
a little race on the answer with you

container cannot be created w/o wallet. Object can: it will be owned by anonymous and imo it's pretty expected, if u wanna own the object - provide wallet